firmware/common2015/drivers/kicker-board/KickerBoard.cpp

```diff
@@ -89,4 +89,61 @@ bool KickerBoard::flash(bool onlyIfDifferent, bool verbose) {
     return true;
 }
 
+bool KickerBoard::send_to_kicker(uint8_t cmd, uint8_t arg, uint8_t& ret_val,
+                                 bool verbose) {
+    chipSelect();
+    // Returns state (charging, not charging), but we don't care about that
+    uint8_t charge_resp = _spi->write(cmd);
+    // Should return the command we just sent
+    uint8_t command_resp = _spi->write(arg);
+    // Should return final response to full cmd, arg pair
+    ret_val = _spi->write(BLANK);
+    chipDeselect();
+
+    bool command_acked = command_resp == cmd;
+    if (verbose) {
+        if (!command_acked) {
+            printf("Kicker failed to ack command! -- ");
+        }
+
+        printf("Kicker: CHG:%02X, CMD:%02X, RET:%02X\r\n", charge_resp,
```

Comment:
Do you know what the behavior of LOG() will be when this is called from one of the hw-test files? The verbosity flags are almost exclusively used within the hw-tests and it would be nice if LOG() just printed the message over Serial like usual.